conda-forge.yml

```diff
@@ -1,9 +1,17 @@
 build_platform:
   osx_arm64: osx_64
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
 conda_build:
   pkg_format: '2'
+conda_build_tool: rattler-build
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+provider:
+  linux_aarch64: azure
+  linux_ppc64le: azure
+shellcheck:
```

Comment:
Wow I didn't know about this. I have some other recipes that could definitely use this.

Reply:
Oooh, yeah, Travis-CI: you sucked so much less than things before you (shudders in Jenkins), but it's basically dead to me (or maybe conda-forge at large) now.